src/explore-education-statistics-common/src/modules/release/components/ReleaseDataAccordion.module.scss

```diff
@@ -3,16 +3,6 @@
 .container {
   margin-bottom: govuk-spacing(9);
   margin-top: -(govuk-spacing(6));
-
-  @include govuk-media-query($until: tablet) {
-    margin-top: 0;
-  }
-
-  @include govuk-media-query($from: tablet) {
-    :global(.govuk-accordion__section-header) {
-      border-top: 0;
-    }
-  }
```

Comment:
These styles are needed for spacing on small screens and so it doesn't look like there's a double width bottom border on the summary box on large screens.

The margin one can safely be added back, I think, and you can change the border one so it only targets the first header with something convoluted like:
```
:global(.govuk-accordion__section:is(:first-child)) {
    :global(.govuk-accordion__section-header) {
      border-top: 0;
    }
  }
```